src/lib.rs

```diff
@@ -55,19 +55,36 @@ const SECTION_LEN: usize = 3; // 1,323.00 <- "323" is a section
 /// Every 100 coins represents a banknote. (coin: 100 => 1.00)
 #[derive(Debug, Clone, Hash, Default, PartialEq, Eq, PartialOrd)]
 pub struct Currency {
-    symbol: Option<char>,
+    symbol: String,
```

Comment:
I would still rather see `Option<String>`. I know `""` is effectively the same in terms of display but `None` is semantically more appropriate in my mind.

Reply:
I'm afraid it would introduce unnecessary burden on the user to unwrap options when it intuitively isn't needed. Are you sure this would be best?